aws/tailers/stack_event.go

```diff
@@ -20,6 +20,16 @@ const (
 	FilterStackEventStatus       = "status"
 	FilterStackEventStatusReason = "reason"
 	FilterStackEventType         = "type"
+
+	// valid stack status codes
+	// http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/using-cfn-describing-stacks.html#w2ab2c15c15c17c11
+	cfStackEventUpdateInProgress = "UPDATE_IN_PROGRESS"
+	cfStackEventCreateInProgress = "CREATE_IN_PROGRESS"
+	cfStackEventDeleteInProgress = "DELETE_IN_PROGRESS"
```

Comment:
Hi, 
Thanks again for your contritbution. 
You should use the aws-sdk-go constants, rather than define your own: `cloudformation.ResourceStatusUpdateInProgress`, `cloudformation.DELETE_IN_PROGRESS`, etc. See [here](https://github.com/wallix/awless/blob/master/vendor/github.com/aws/aws-sdk-go/service/cloudformation/api.go#L11255) for all constants.